src/realm.h

```diff
@@ -2585,6 +2585,11 @@ RLM_API void realm_app_config_set_default_request_timeout(realm_app_config_t*, u
 RLM_API void realm_app_config_set_platform(realm_app_config_t*, const char*) RLM_API_NOEXCEPT;
 RLM_API void realm_app_config_set_platform_version(realm_app_config_t*, const char*) RLM_API_NOEXCEPT;
 RLM_API void realm_app_config_set_sdk_version(realm_app_config_t*, const char*) RLM_API_NOEXCEPT;
+/**
+ * Get an existing @a realm_app_credentials_t and return it's json representation
+ * @return: a non-null ptr to the string representing the json configuration
```

Comment:
make a note that it is the responsibility of the caller to free the returned pointer with realm_free()